pkg/ddevapp/config.go

```diff
@@ -1206,7 +1209,29 @@ func WriteBuildDockerfile(app *DdevApp, fullpath string, userDockerfilePath stri
 	// Normal starting content is the arg and base image
 	contents := `
 #ddev-generated - Do not modify this file; your modifications will be overwritten.
+`
+	// If there are user stage.Dockerfile.* files, insert their contents
+	if userDockerfilePath != "" {
+		files, err := filepath.Glob(filepath.Join(userDockerfilePath, "stage.Dockerfile.*"))
```

Comment:
Docs say:

* `.ddev/web-build/stage.Dockerfile.*`
* `.ddev/web-build/stage.Dockerfile`
* `.ddev/db-build/stage.Dockerfile.*`
* `.ddev/db-build/stage.Dockerfile`

But this code doesn't support `stage.Dockerfile` variant without `.*`

Reply:
nice catch! Fixing and will add test for this and for pre.Dockerfile which was not tested (the pre.Dockerfile - no suffix - variant)